From: Mohammed Sadiq <sadiq@sadiqpk.org>
Date: Thu, 19 Sep 2019 20:44:12 +0530
Subject: [PATCH 2/4] wifi: Use new connection editor

---
 panels/network/net-device-wifi.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/panels/network/net-device-wifi.c b/panels/network/net-device-wifi.c
index 69fe2e1..7065e0e 100644
--- a/panels/network/net-device-wifi.c
+++ b/panels/network/net-device-wifi.c
@@ -962,6 +962,8 @@ ap_sort (gconstpointer a, gconstpointer b, gpointer data)
         return 0;
 }
 
+#include "connection-editor/cc-connection-editor.h"
+#include "pureos.h"
 static void
 show_details_for_row (NetDeviceWifi *self, CcWifiConnectionRow *row, CcWifiConnectionList *list)
 {
@@ -972,6 +974,7 @@ show_details_for_row (NetDeviceWifi *self, CcWifiConnectionRow *row, CcWifiConne
         connection = cc_wifi_connection_row_get_connection (row);
         ap = cc_wifi_connection_row_best_access_point (row);
 
+#include "use-new-editor.c"
         editor = net_connection_editor_new (connection, self->device, ap, self->client);
         gtk_window_set_transient_for (GTK_WINDOW (editor), GTK_WINDOW (gtk_widget_get_toplevel (GTK_WIDGET (row))));
         gtk_window_present (GTK_WINDOW (editor));
